src/weatherreport/src/weatherreport_check_memory_use.erl

```diff
@@ -0,0 +1,66 @@
+%% -------------------------------------------------------------------
+%%
+%% riaknostic - automated diagnostic tools for Riak
```

Comment:
Another riaknostic header that probably shouldn't be here.

Reply:
I suspect it should based on the list I compiled.